Cargo.toml

```diff
@@ -12,17 +12,15 @@ license = "MIT"
 
 [dependencies]
 flate2 = "0.2"
-hyperlocal = "0.2"
+hyperlocal = { git = "https://github.com/abronan/hyperlocal.git", branch = "update_hyper" }
```

Comment:
Will update this dependency (which currently points to my fork) if a new version is tagged or to the master branch of the main repository :)